cmd/main.go

```diff
@@ -531,23 +524,6 @@ func uploadBootFiles(objectHandler s3wrapper.API, openshiftVersionsMap models.Op
 	return errs.Wait()
 }
 
-func newISOInstallConfigGenerator(log *logrus.Entry, objectHandler s3wrapper.API, operatorsApi operators.API) generator.ISOInstallConfigGenerator {
-	var configGenerator generator.ISOInstallConfigGenerator
-	switch Options.Storage {
-	case storage_s3:
-		kclient, err := client.New(config.GetConfigOrDie(), client.Options{Scheme: scheme.Scheme})
```

Comment:
Is it alright that I don't see `kclient` anywhere in the new `generator` code?

Reply:
Yeah, that was part of the goal. We don't talk to the cluster during this process anymore.